Source/Core/VideoCommon/PostProcessing.cpp

```diff
@@ -435,81 +478,199 @@ void PostProcessing::BlitFromTexture(const MathUtil::Rectangle<int>& dst,
     RecompilePipeline();
   }
 
-  if (!m_pipeline)
-    return;
+  // By default all source layers will be copied into the respective target layers
+  const bool copy_all_layers = src_layer < 0;
+  src_layer = std::max(src_layer, 0);
 
-  FillUniformBuffer(src, src_tex, src_layer);
-  g_vertex_manager->UploadUtilityUniforms(m_uniform_staging_buffer.data(),
-                                          static_cast<u32>(m_uniform_staging_buffer.size()));
-
-  g_gfx->SetViewportAndScissor(
-      g_gfx->ConvertFramebufferRectangle(dst, g_gfx->GetCurrentFramebuffer()));
-  g_gfx->SetPipeline(m_pipeline.get());
-  g_gfx->SetTexture(0, src_tex);
+  MathUtil::Rectangle<int> src_rect = src;
   g_gfx->SetSamplerState(0, RenderState::GetLinearSamplerState());
```

Comment:
Why did you double the amount of textures used?

Reply:
Ah, I forgot to delete that after scrapping some work on improving texture samplings that I decide to delay to a follow up PR.